components/Newsfeed/Newsfeed.tsx

```diff
@@ -0,0 +1,276 @@
+import {
+  InstantSearch,
+  Pagination
+} from "@alexjball/react-instantsearch-hooks-web"
+import { collection, query, where, getDocs } from "firebase/firestore"
+import ErrorPage from "next/error"
+import { useCallback, useEffect, useState } from "react"
+import TypesenseInstantSearchAdapter from "typesense-instantsearch-adapter"
+import { useMediaQuery } from "usehooks-ts"
+import { useAuth } from "../auth"
+import { Col, Row, Spinner } from "../bootstrap"
+import { usePublicProfile } from "../db"
+import { firestore } from "../firebase"
+import {
+  Header,
+  HeaderTitle,
+  StyledContainer
+} from "./StyledNewsfeedComponents"
+import { AlertCard } from "components/AlertCard/AlertCard"
+import { getServerConfig } from "components/search/common"
+import { SearchContainer } from "components/search/SearchContainer"
+import { SearchErrorBoundary } from "components/search/SearchErrorBoundary"
+import { initialSortByValue } from "components/search/SortBy"
+import { currentGeneralCourt } from "functions/src/shared"
+
+type ElementProps = {
+  bodyText: string
+  court: string
+  header: string
+  id: string
+  subheader: string
+  timestamp: string
+  topicName: string
+  type: string
+}
+
+type Props = ElementProps[]
```

Comment:
I'm curious about these `Props` and `ElementProps` names... perhaps we could name it something more specific to the page, like `NewsBites`, `NewsUpdates`, `TopicUpdates`, or `TopicNews`? I find trying to name the data we're working with as specific to the kinds of things users call it as possible can help readability of the code. If there's a consistent name we call this data within mockups or specs, we should use that.

Reply:
Looking through [the initial spec](https://docs.google.com/document/d/1C9iFYSmj4az-03wAoq-ZaRzKquh-znUFxRGizGi9yRU/edit#heading=h.s52jir1mj0j5) it looks like these are `Notifications`?

Reply:
switched Props to Notifications and ElementProps to NotificationProps